src/main/entities/CustomPassword.java

```diff
@@ -0,0 +1,16 @@
+package entities;
+
+public class CustomPassword extends Password {
+    StrengthChecker strengthChecker = new StrengthChecker();
+    public void checkStrength() {
+        strengthChecker.computeStrength(passwordValue);
+    }
+
+    public Strength getStrength() {
+        return strengthChecker.strength;
+    }
+
+    public String getSuggestion() {
+        return strengthChecker.suggestion.getSuggestion();
```

Comment:
Should probably use a getter instead of directly accessing .suggestion.

Reply:
Yeah `suggestion` is just a class variable and we use getter on that.